build.gradle

```diff
@@ -77,6 +77,9 @@ configure(subprojects) {
         mavenCentral()
         maven {
             url 'https://storage.googleapis.com/android-ci/mvn/'
+            metadataSources {
+                artifact()
+            }
```

Comment:
I think you need to do the same in buildSrc tests to make tests pass:
https://github.com/firebase/firebase-android-sdk/blob/13083a2f991b81059ca3fe05764908b38c3d077b/buildSrc/src/test/kotlin/com/google/firebase/gradle/plugins/PublishingPluginTests.kt#L339-L341

Reply:
Hmm, lets find out :')

Also, should we add `--stacktrace` to the buildSrc action?

Reply:
Sure enough, that fixed it:)

Reply:
I don't think `--stacktrace` would be useful in this case, the problem was with tests failing and these failures can be seen in the build log.